zigpy/endpoint.py

```diff
@@ -133,7 +133,7 @@ def add_output_cluster(self, cluster_id, cluster=None):
             if isinstance(value, bytes):
                 try:
                     value = value.split(b'\x00')[0]
-                    attributes[key] = value.decode('ascii').strip()
+                    attributes[key] = value.decode().strip()
                 except UnicodeDecodeError:
                     # Unsure what the best behaviour here is. Unset the key?
                     pass
```

Comment:
in case of an UnicodeDecodeError exception, should we set the key to "hexlified" bytes?

Reply:
Yes, that’s done in several spots in bellows already. Makes sense to do it here as well.